README.md

```diff
@@ -1,3 +1,26 @@
 # police-data-trust
+# Installation
+
+1. Make sure that you have [Python 3.8](https://www.python.org/), [Git](https://git-scm.com/), and [PostgreSQL](https://www.postgresql.org/) are installed on your machine.
```

Comment:
"Make sure that ... are installed" or alternatively "Make sure that you have ... installed"